concent_api/utils/constants.py

```diff
@@ -0,0 +1,52 @@
+    MESSAGE_FILES_CHECKSUM_EMPTY                            = 'message.files.checksum.empty'
+    MESSAGE_FILES_CHECKSUM_INVALID_SHA1_HASH                = 'message.files.checksum.invalid_sha1_hash'
+    MESSAGE_FILES_CHECKSUM_UNRECOGNIZED_TYPE                = 'message.files.checksum.unrecognized_type'
+    MESSAGE_FILES_CHECKSUM_WRONG_FORMAT                     = 'message.files.checksum.wrong_format'
+    MESSAGE_FILES_CHECKSUM_WRONG_TYPE                       = 'message.files.checksum.wrong_type'
+    MESSAGE_FILES_PATH_NOT_LISTED_IN_FILES                  = 'message.files.path.not_listed_in_files'
+    MESSAGE_FILES_PATH_NOT_UNIQUE                           = 'message.files.path.not_unique'
+    MESSAGE_FILES_SIZE_NEGATIVE                             = 'message.files.size.negative'
+    MESSAGE_FILES_SIZE_WRONG_TYPE                           = 'message.files.size.wrong_type'
+    MESSAGE_FILES_WRONG_TYPE                                = 'message.files.wrong_type'
+    MESSAGE_OPERATION_INVALID                               = 'message.operation.invalid'
+    MESSAGE_STORAGE_CLUSTER_INVALID_URL                     = 'message.storage_cluster_invalid_url'
+    MESSAGE_STORAGE_CLUSTER_UNKNOWN                         = 'message.storage_cluster_unknown'
+    MESSAGE_STORAGE_CLUSTER_WRONG_TYPE                      = 'message.storage_cluster_wrong_type'
+    MESSAGE_TASK_ID_BLANK                                   = 'message.task_id.blank'
+    MESSAGE_TASK_ID_WRONG_LENGTH                            = 'message.task_id.wrong_length'
+    MESSAGE_TASK_ID_WRONG_TYPE                              = 'message.task_id.wrong_type'
+    MESSAGE_TIMESTAMP_ERROR                                 = 'message.timestamp.error'
+    MESSAGE_TIMESTAMP_TOO_FAR_IN_FUTURE                     = 'message.timestamp.too_far_in_future'
+    MESSAGE_TIMESTAMP_TOO_OLD                               = 'message.timestamp.to_old'
+    MESSAGE_TOKEN_EXPIRATION_DEADLINE_PASSED                = 'message.token_expiration_deadline_passed'
+    MESSAGE_TOKEN_EXPIRATION_DEADLINE_WRONG_TYPE            = 'message.token_expiration_deadline_wrong_type'
+    MESSAGE_VALUE_NOT_INTEGER                               = 'message.value_not_integer'
+    MESSAGE_UNABLE_TO_DESERIALIZE                           = 'message.unable_to_deserialize'
+    MESSAGE_UNEXPECTED                                      = 'message.unexpected'
+    MESSAGE_UNKNOWN                                         = 'message.unknown'
+    MESSAGE_WRONG_FIELDS                                    = 'message.wrong_fields'
+    QUEUE_COMMUNICATION_NOT_STARTED                         = 'queue.communication_not_started'
+    QUEUE_MESSAGE_ALREADY_PROCESSED                         = 'queue.message_already_processed'
+    QUEUE_RESPONSE_ALREADY_SUBMITTED                        = 'queue.response_already_submitted'
+    QUEUE_TIMEOUT                                           = 'queue.timeout'
```

Comment:
Not all codes are used in the code. Please leave only the ones that are that in subsequent pull requests it's easy to see which ones you're adding.

It's hard to say if these codes will be a good idea without seeing how they're used first.

Reply:
:+1: